iOS_SDK/OneSignalSDK/Source/OneSignalHelper.m

```diff
@@ -522,7 +522,7 @@ + (BOOL) canGetNotificationTypes {
     return [self isIOSVersionGreaterThanOrEqual:@"8.0"];
```

Comment:
this can be
`return [self isIOSVersionGreaterThanOrEqual:@"9.0"];`?

Reply:
This can actually just be removed since our min version is iOS 9 this essentially says return True. Removed in latest commit